datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

```diff
@@ -156,8 +156,8 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
         datasetCreator.name);
   }
 
-  public void userJourney(TestUserSpecification testUser) throws Exception {
-    ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
+  public void userJourney() throws Exception {
+    ApiClient apiClient = DataRepoUtils.getClientForTestUser(datasetCreator, server);
```

Comment:
...though I am surprised since I thought all of the users got permissions on the snapshot